docs/config/kowl-business.yaml

```
@@ -44,6 +44,7 @@ kafka:
   #     username:
   #     password: # can be set via the --kafka.sasl.gssapi.password flag as well
   #     realm:
+  #     enableFast: True
```

```suggestion
  #     enableFast: true
```

Just for consistency :)